From 7b50367b2715881a973e6bfab99af68dd50396ed Mon Sep 17 00:00:00 2001
From: Devin Lin <espidev@gmail.com>
Date: Wed, 22 Jun 2022 11:48:46 -0400
Subject: [PATCH] components: Attempt to fix some cyclic dependencies

(cherry picked from commit 0deef11ba72fd22ad0048c9f048787fbe43dfd4b)
---
 components/mobileshell/qml/Shell.qml              | 2 --
 components/mobileshell/qml/components/Haptics.qml | 1 -
 2 files changed, 3 deletions(-)

diff --git a/components/mobileshell/qml/Shell.qml b/components/mobileshell/qml/Shell.qml
index 522e56f0e..6d476a39f 100644
--- a/components/mobileshell/qml/Shell.qml
+++ b/components/mobileshell/qml/Shell.qml
@@ -7,8 +7,6 @@
 import QtQuick 2.15
 import QtQuick.Window 2.15
 
-import org.kde.plasma.private.mobileshell 1.0 as MobileShell
-
 pragma Singleton
 
 /**
diff --git a/components/mobileshell/qml/components/Haptics.qml b/components/mobileshell/qml/components/Haptics.qml
index 9bfec9527..4a2a4028d 100644
--- a/components/mobileshell/qml/components/Haptics.qml
+++ b/components/mobileshell/qml/components/Haptics.qml
@@ -4,7 +4,6 @@
  */
 
 import QtQuick 2.15
-import QtFeedback 5.0
 
 import org.kde.plasma.private.mobileshell 1.0 as MobileShell
 
-- 
2.36.1

